frontend/src/app/coworking/reservation/reservation.service.ts

```diff
@@ -58,6 +58,18 @@ export class ReservationService {
     );
   }
 
+  checkin(reservation: Reservation) {
+    let endpoint = `/api/coworking/reservation/${reservation.id}`;
+    let payload = { id: reservation.id, state: 'CHECKED_IN' };
```

Comment:
Nit: Would be good to use an `enum` rather than the string literal `'CHECKED_IN'`. This might be out of scope for this branch though